0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -124,34 +83,23 @@ The response MUST contain at least the following headers:
 | `Content-Type`  | MUST be `application/spsp4+json` to indicates the response is encoded as [JSON](http://www.json.org/) and that the ILP payment should be sent via STREAM. |
 | `Cache-Control` | Indicates how long the SPSP Client should cache the response. See supported cache-control directives below. |
 
-To handle as many transactions per second as possible, the SPSP Client caches results from the SPSP Server. The information returned by the SPSP Server is not expected to change rapidly, so repeated requests for the same information are usually redundant. The server communicates how long to cache results for using the HTTP-standard [`Cache-Control` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) in the responses to RESTful API calls.
+To handle as many transactions per second as possible, the SPSP Client caches results from the SPSP Server. The information returned by the SPSP Server is not expected to change rapidly, so repeated requests for the same information are usually redundant. The SPSP Server communicates how long to cache results for using the HTTP-standard [`Cache-Control` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) in the responses to RESTful API calls.
```

Comment:
I actually think you can't reuse STREAM parameters, so we should remove this section about caching and say that the SPSP response MUST NOT be cached.

@emschwartz is that correct? If you reuse the same STREAM parameters after the connection has closed (or while it is still open) will that give you an error?

Reply:
@emschwartz ^

Reply:
If I understand correctly, we could keep the cache if this spec does not mention sending data. What do you prefer @sharafian @emschwartz ?

Reply:
I guess if the receiver is stateless and doesn't care then it's fine to say they can set cache control. They'll know whether they accept protocols on top of SPSP that use data. So I'd still keep the bit in this spec about how to do data over SPSP and I'd put the cache control headers as a MAY. Maybe include a note that says "SPSP details should not be cached except in special scenarios" so that nobody accidentally includes them and gets mystifying bugs?

Reply:
@emschwartz can you explain this a bit:

> If you want to use STREAM to send data as well, then caching the results would be a bad idea. The receiver would likely get confused about the order of the packets.

I'd assume that if a client attempts to create a connection using the same parameters the second connection attempt would fail while the first connection is still open?

Is the correct behaviour defined in the STREAM spec?